src/management-system-v2/components/tabcard-model-metadata.tsx

```diff
@@ -103,12 +111,35 @@ const TabCard: FC<TabCardProps> = ({ item, selection, setSelection, tabcard, com
     setActiveTabKey(key);
   };
 
+  const clipAndHighlightText = useCallback(
+    (dataIndexElement, record, index) => {
+      const withoutSearchTerm = dataIndexElement?.split(search);
```

Comment:
Fuse has a `includeMatches` option to get the matched characters

Reply:
I'd rather do this in a separate pull request, see [here](https://github.com/PROCEED-Labs/proceed/pull/135/files/aa7060145c04367a274bf54b991dc874d4966554#r1371851031).